src/fighter_state.rs

```diff
@@ -1011,6 +1054,60 @@ fn ground_slam(
     }
 }
 
+fn bomb_throw(
+    mut commands: Commands,
+    mut fighters: Query<
+        (
+            &mut Animation,
+            &mut LinearVelocity,
+            &Facing,
+            &Transform,
+            &Handle<FighterMeta>,
+            &mut BossBombThrow,
+        ),
+        With<Boss>,
+    >,
+    fighter_assets: Res<Assets<FighterMeta>>,
+) {
+    for (mut animation, mut velocity, facing, transform, meta_handle, mut ground_slam) in
+        &mut fighters
+    {
+        // Start the attack
+        if let Some(fighter) = fighter_assets.get(meta_handle) {
+            let mut offset = fighter.attack.hitbox.offset;
+            if facing.is_left() {
+                offset.x *= -1.0
+            }
+            offset.y += fighter.collision_offset;
+            let attack_frames = fighter.attack.frames;
```

Comment:
Instead of using the attack.frames metadata now we are using the attack.frames from the last attack on `AvailableAttacks`